controllers/checlusterrestore/backup_data_restorer.go

```diff
@@ -28,7 +28,7 @@ import (
 	"github.com/sirupsen/logrus"
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
-	extv1beta1 "k8s.io/api/extensions/v1beta1"
+	networking "k8s.io/api/networking/v1"
```

Comment:
To match the other names
```suggestion
	networkingv1 "k8s.io/api/networking/v1"
```